app/routes.py

```diff
@@ -1 +1,305 @@
-from flask import Blueprint
\ No newline at end of file
+from app import db
+from app.models.goal import Goal
+from app.models.task import Task
+from datetime import datetime
+from flask import Blueprint, jsonify, make_response, request, abort
+from .routes_helper import get_one_valid_id
+import requests
+
+tasks_bp = Blueprint("tasks", __name__, url_prefix="/tasks")
+#------------------------------------WAVE 1----------------------------------
+#Create a Task: Valid Task With null completed_at
+@tasks_bp.route("", methods=["POST"])
+def create_task():
+    request_body = request.get_json()
+    #Create a Task: Invalid Task With Missing Data
+    if  "title" not in request_body or\
+        "description" not in request_body:
+            abort(make_response({"details": "Invalid data"},400))
```

Comment:
It's good that you're doing input validation here, but it would be better to let the end user know what is invalid.